src/main/java/com/bootcamp_w3_g3/repository/CompradorRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.bootcamp_w3_g3.repository;
+
+import com.bootcamp_w3_g3.model.entity.Comprador;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+/**
+ * @author hugo damm
+ */
+
+@Repository
+public interface CompradorRepository extends JpaRepository<Comprador, Long> {
+
+    Comprador getById (Long id);
```

Comment:
Hugo troque este método get por findByUsuario(String usuario)
```suggestion
    Comprador findByUsuario(String usuario));
```